js/tests/launchAsync.test.ts

```diff
@@ -217,6 +217,31 @@ describe('launchAsync tests', () => {
         expect(iframe.getAttribute('allowfullscreen')).toBeNull();
     });
 
+    it('launches with onExit callback', async () => {
+        jest.useRealTimers();
+        expect.assertions(1);
+
+        const cbOnExit = jest.fn(() => { });
+
+        const options: Options = { onExit: () => { cbOnExit(); } };
+        const launchPromise = launchAsync(SampleToken, null, SampleContent, options);
+        window.postMessage('ImmersiveReader-LaunchSuccessful', '*');
+
+        await launchPromise;
+
+        window.postMessage('ImmersiveReader-Exit', '*');
+
+        const timeout = (ms: number) => new Promise(resolve => { setTimeout(resolve, ms); });
```

Comment:
can you condense the timeout code to something like the following?

    await new Promise(resolve => { setTimeout(resolve, 1); });

and add comments that this is to yield this thread of execution to allow the exit message to get processed

Reply:
i have condense the timeout code and added the comment